easybuild/easyconfigs/f/fftlib/fftlib-20170628-gompi-2020b.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'MakeCp'
+
+name = 'fftlib'
+version = '20170628'
+_commit = '44c5fa0'
+
+homepage = 'https://github.com/flwende/fftlib'
+description = """A library that intercepts FFTW calls and adds features on top of it.
+ In particular, it enables FFT plan reuse when there are multiple calls for the same geometry."""
+
+toolchain = {'name': 'gompi', 'version': '2020b'}
+toolchainopts = {'openmp': True, 'cstd': 'c++11'}
+
+source_urls = ['https://github.com/flwende/fftlib/archive/']
+sources = [{'filename': '%s.zip' % _commit, 'extract_cmd': 'unzip %s'}]
```

Comment:
Please rename on download:

```python
sources = [{'download_filename': '%s.zip' % local_commit, 'filename': SOURCE_ZIP}]
```

Why is the custom extract command required?

Reply:
my bad (I had started with `CmdCp` easyblock and then realized `MakeCp` with a custom `build_cmd` was more appropriate)